src/applications/personalization/profile-2/tests/unit-test-helpers.js

```diff
@@ -6,6 +6,22 @@ import profile from 'applications/personalization/profile360/reducers';
 import connectedApps from 'applications/personalization/profile-2/components/connected-apps/reducers/connectedApps';
 import profileUi from '../reducers';
 
+// returns a Promise that resolves if the cb does not throw an error when it's
+// run after the given timeout. The Promise is rejected if the cb throws an
+// error when it runs.
+export function waitAndCheck(cb, timeout) {
```

Comment:
In an upcoming PR I will use this helper instead of the `elementNotRemoved()` helper, but I want to get feedback on my first draft of this helper before I do that.